packages/react-tinacms-github/README.md

```diff
@@ -28,7 +28,12 @@ const REPO_FULL_NAME = process.env.REPO_FULL_NAME // e.g: tinacms/tinacms.org
 
 const cms = new TinaCMS({
   apis: {
-      github: new GithubClient('/api/proxy-github', REPO_FULL_NAME),
+    github: new GithubClient({
+      proxy: '/api/proxy-github',
+      clientId: process.env.GITHUB_CLIENT_ID,
+      authCallbackRoute: '/api/create-github-access-token'
+      baseRepoFullName: REPO_FULL_NAME,
```

Comment:
Seems inconsistent for no real reason to have GITHUB_CLIENT_ID be pulled directly from the env while assigning REPO_FULL_NAME to a local var.